atomic_red_team/test_data/float_type.yaml

```diff
@@ -0,0 +1,18 @@
+attack_technique: T1003
```

Comment:
`atomic_red_team/tests` would contain all the test cases for code within `atomic_red_team/**/*.py`